config/environments/development.rb

```diff
@@ -12,12 +12,23 @@
   # Show full error reports.
   config.consider_all_requests_local = true
 
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    :user_name => '0af7a3b580e9e0',
+    :password => 'fb07158652bb0d',
```

Comment:
NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO!!!!!!!!!!!!!
😨 😨 😨 😨 😠 😡 
You can not do that!!!! It's clearly specify in the card: `Las credenciales nunca se tienen que subir a github`